tests/src/test/java/alluxio/master/lineage/LineageMasterClientRestApiTest.java

```diff
@@ -66,10 +59,7 @@ public void before() throws Exception {
     Whitebox.setInternalState(alluxioMaster, "mLineageMaster", mLineageMaster);
     mHostname = mResource.get().getHostname();
```

Comment:
I can imagine that in some cases the hostname might be different from the local Alluxio cluster, so I would prefer to keep that out of the RestApiTest